src/emqx_frame.erl

```diff
@@ -473,10 +473,14 @@ serialize_variable(#mqtt_packet_unsubscribe{packet_id     = PacketId,
 serialize_variable(#mqtt_packet_unsuback{packet_id    = PacketId,
                                          properties   = Properties,
                                          reason_codes = ReasonCodes},
```

Comment:
Will this `ReasonCodes` be filled by some value or it is always `undefined` when mqtt version < 5 ?

Reply:
it should be undefine when mqtt version < 5